polling_stations/apps/uk_geo_utils/helpers.py

```diff
@@ -16,6 +16,29 @@ def get_onsud_model():
     return get_model('ONSUD_MODEL', 'uk_geo_utils.Onsud')
 
 
+class Postcode:
```

Comment:
I have some general questions about the scope of `Postcode`. Obviously what it does at the moment is fairly simple, but should it work out if an input is valid or not? Should it raise on instantiation with an invalid postcode?

If it attempts to validate, should it just work out the format, or existence?

Reply:
Ah, I see in a later commit you've introduced one check that raises an generic `ValueError`.

Reply:
Sorry, ignore this – I've seen you wrote about this in the description :/

Reply:
That's exactly what I'm getting at in the first post of this PR in the section

> The other thing this abstraction gives us
...
> but I think its useful to create the right structure to slot these things into as I refactor